x/feegrant/internal/types/key.go

```diff
@@ -0,0 +1,37 @@
+package types
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+const (
+	// ModuleName is the module name constant used in many places
+	ModuleName = "feegrant"
+
+	// StoreKey is the store key string for supply
+	StoreKey = ModuleName
+
+	// RouterKey is the message route for supply
+	RouterKey = ModuleName
+
+	// QuerierRoute is the querier route for supply
+	QuerierRoute = ModuleName
+)
+
+var (
+	// FeeAllowanceKeyPrefix is the set of the kvstore for fee allowance data
+	FeeAllowanceKeyPrefix = []byte{0x00}
+)
+
+// FeeAllowanceKey is the canonical key to store a grant from granter to grantee
+// We store by grantee first to allow searching by everyone who granted to you
+func FeeAllowanceKey(granter sdk.AccAddress, grantee sdk.AccAddress) []byte {
```

Comment:
see previous note in the keeper file.

Reply:
I cannot see this note in the review - there are so many notes they all get collapsed by the webui. Can you please reference the note (link or just cut-paste the comment)?